packages/@uppy/xhr-upload/src/index.ts

```diff
@@ -194,13 +199,16 @@ export default class XHRUpload<
      */
     this.#getFetcher = (files: UppyFile<M, B>[]) => {
       return async (
-        url: Parameters<typeof fetcher>[0],
-        options: NonNullable<Parameters<typeof fetcher>[1]>,
+        url: string,
```

Comment:
Why this change?

Reply:
Because in this plugin we expect `url` to be a string, regardless of what fetcher supports so it's better to match it to the plugin code. `fetcher` also expects a `string` and should it ever change in `fetcher` we still get a type error.

Reply:
Where? AFAICT `url` is only passed to `fetcher`, so it only matters what `fetcher` supports IMO

Reply:
No, it only matters what this type supports:

https://github.com/transloadit/uppy/blob/4442722c8ae6577b8e4a8a7b952e88b0ad3973ef/packages/%40uppy/xhr-upload/src/index.ts#L30

So it's either `string` or `Opts<M, B>['endpoint']`. But it's such I minor thing I wouldn't overthink it.

Reply:
I guess it only matters if `fetcher` has a change in its signature where it no longer supports `string`, but we'd get a type error anyway.